docker/encfs/encfs.sh

```diff
@@ -22,11 +22,56 @@ fi
 
 echo CmdlineArgs = $CmdlineArgs
 
+cleanup_unmount() {
+  mount_array=$(process_encoded_string $EncfsSideCarArgs)
+  echo "Cleaning up by unmounting the following remote filesystem folder."
+  for mount_point in "${mount_array[@]}"; do
+    echo "attempting to unmount $mount_point"
+    output=$(umount $mount_point 2>&1)
```

Comment:
why not `umount $mount_point 2>&1` and then just check the exit status `if [ $? -eq 0 ]`. If the first command has any output, it should automatically print it out I would think and then there's no need for extra variable assignments...